sw/device/tests/dif/dif_plic_unittest.cc

```diff
@@ -112,7 +112,7 @@ class IrqTest : public PlicTest {
           {RV_PLIC_IE0_2_REG_OFFSET, RV_PLIC_IE0_2_E_95_BIT},
           {RV_PLIC_IE0_3_REG_OFFSET, RV_PLIC_IE0_3_E_127_BIT},
           {RV_PLIC_IE0_4_REG_OFFSET, RV_PLIC_IE0_4_E_159_BIT},
-          {RV_PLIC_IE0_5_REG_OFFSET, RV_PLIC_IE0_5_E_176_BIT},
+          {RV_PLIC_IE0_5_REG_OFFSET, RV_PLIC_IE0_5_E_177_BIT},
```

Comment:
We really, really, really need to parameterize this this test so that we're not relying on the PLIC instantiation so strongly. @silvestrst

Reply:
I will have a look when have time.